docs/kubernetes/operator/instrumenting-nodejs.md

```diff
@@ -0,0 +1,163 @@
+
+    ```bash
+    $ kubectl describe pod nodejs-app-8d84c47b8-8h5z2 -n nodejs-ns
+    Name:             nodejs-app-8d84c47b8-8h5z2
+    Namespace:        nodejs-ns
+    ...
+    ...
+    Init Containers:
+      opentelemetry-auto-instrumentation-nodejs:
+        Container ID:  containerd://cbf67d7ca1bd62c25614b905a11e81405bed6fd215f2df21f84b90fd0279230b
+        Image:         docker.elastic.co/observability/elastic-otel-nodejs:1.0.0
+        Image ID:      docker.elastic.co/observability/elastic-otel-nodejs@sha256:28d65d04a329c8d5545ed579d6c17f0d74800b7b1c5875e75e0efd29e210566a
+        Port:          <none>
+        Host Port:     <none>
+        Command:
+          cp
+          /nodejsagent.jar
+          /otel-auto-instrumentation-nodejs/nodejsagent.jar
+        State:          Terminated
+          Reason:       Completed
+          Exit Code:    0
+          Started:      Wed, 13 Nov 2024 15:47:02 +0100
+          Finished:     Wed, 13 Nov 2024 15:47:03 +0100
+        Ready:          True
+        Restart Count:  0
+        Environment:    <none>
+        Mounts:
+          /otel-auto-instrumentation-nodejs from opentelemetry-auto-instrumentation-nodejs (rw)
+          /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-swhn5 (ro)
+    ```
+
+    - The main container of the deployment is using the SDK as `nodejsagent`:
+
+    ```bash
+    ...
+    Containers:
+      nodejs-app:
+        Environment:
+    ...
+          JAVA_TOOL_OPTIONS:                      -nodejsagent:/otel-auto-instrumentation-nodejs/nodejsagent.jar
+          OTEL_SERVICE_NAME:                     nodejs-app
+          OTEL_EXPORTER_OTLP_ENDPOINT:           http://opentelemetry-kube-stack-daemon-collector.opentelemetry-operator-system.svc.cluster.local:4318
+    ...
+    ```
+
+    - The Pod has an `EmptyDir` volume named `opentelemetry-auto-instrumentation-nodejs` mounted in both the main and the init containers in path `/otel-auto-instrumentation-nodejs`.
+
+    ```bash
+    Init Containers:
+      opentelemetry-auto-instrumentation-nodejs:
+    ...
+        Mounts:
+          /otel-auto-instrumentation-nodejs from opentelemetry-auto-instrumentation-nodejs (rw)
+    Containers:
+      nodejs-app:
+    ...
+        Mounts:
+          /otel-auto-instrumentation-nodejs from opentelemetry-auto-instrumentation-nodejs (rw)
+    ...
+    Volumes:
+    ...
+      opentelemetry-auto-instrumentation-nodejs:
+        Type:        EmptyDir (a temporary directory that shares a pod's lifetime)
+    ```
+
+    Ensure the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` points to a valid endpoint and there's network communication between the Pod and the endpoint.
+
+**END OF REVIEW NEEDED SECTION**
+
+5. Confirm data is flowing to **Kibana**:
+
+    - Open **Observability** -> **Applications** -> **Service inventory**, and determine if:
+      - The application appears in the list of services (`nodejs-app` in the example).
```

Comment:
@mdbirnstiehl , @trentm : does the parenthetical statement makes sense here? or would you elaborate it in a different way?

Maybe directly what @trentm suggested in a different thread?
> The application `nodejs-app` appears in the list of services.

Reply:
Looks fine as you have it.